core/src/main/java/org/infinispan/configuration/cache/IndexingConfigurationBuilder.java

```diff
@@ -132,6 +151,7 @@ public IndexingConfigurationBuilder index(Index index) {
     * @return <code>this</code>, for method chaining
     */
    public IndexingConfigurationBuilder autoConfig(boolean autoConfig) {
+      enable();
```

Comment:
Enabling indexing even if autoConfig == false? Maybe this auto-enabling should happen only if autoConfig==true?

Reply:
This is wrong indeed...Guess nobody ever called autoConfig(false) before anywhere